### PR TITLE
react-intl and English, Spanish, and French added for screen texts

### PR DESCRIPTION
**Pull request by frewinchristopher**
As promised, I've internationalized the app and have added English, Spanish, and French!

Note that the tests didn't pass, but this was due to `window.moment is not a function`, and not relevant to the `react-intl` code I added.

Some questions:

- The time format on the login screen is still the same - do we want to change it from 12 hour or 24 hour clock based on locale? That may be a bit tricky; it could only be done if the country was passed in the locale (this isn't always the case with browser locale, i'm not sure if the lightDM api guarantees a full language + country locale)
- Have I missed any major texts that should be translated? If so, they need to be converted using a `<FormattedMessage/>` component (a standard component from the `react-intl` package)

Other notes:

- The three languages (currently) are defined in `messages.js`, I believe `messages.js` is the convention for `react-intl`. Calling the `flattenMessages` function is also a standard for the `react-intl` library, for when you want your `messages.js` to have nested structure.
- It seems that those `moment` `requires()` have to be in index.js. If they are in the components themselves, calling moment.locale(_whatever_) seems to not work and/or overwrite the passed props `locale`
- Note that the `locale` determined in `index.js` is passed down to both components as a prop, as it is needed in the call to moment.locale() (that might also be needed to be played with, while the date will be in the locale language, it may not be in perfect format for such a display (?) )
- The default fallback locale is simple "en" without any "_US" or "_UK" or etc., just plain English.

### CODE REVIEW

File: src/messages.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    LoginScreen: {
+      Hibernate: 'Hibernate',
+      Suspend: 'Suspend',
+      Restart: 'Restart',
+      Shutdown: 'Shutdown',
+      Lock: 'Lock',
+      UserPrompt: 'User'
+    }
+  },
+  'es': {
```
Comment (felipemarinho97) [inline src/messages.js]: This is portuguese :disappointed_relieved:

**Comment by felipemarinho97** [comment]
Thank you so much for the contribution @frewinchristopher ! :smiley: 

Answering the question about the clock format.. it might be best to let the user configure the format in future. Also, did you forget to translate the password input placehokder? :laughing: 

The moment.js dependency is injected by the `lightdm-webkit` and is already localized to the user language.. it seems that these moment.locale workarounds are only needed for working fine in the browser demo. 

Overall, the PR looks great and works properly.

**Comment by frewinchristopher** [comment]
Attempted adding Portuguese and fixing the Spanish translations... probably failed 😂 Note that the password placeholder is also internationalized now and that has been added to each of the languages 👍

**Comment by felipemarinho97** [comment]
I can review the Spanish later, thank you again!

**Pull request merged**